torchdata/nodes/samplers/stop_criteria.py

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+
+class StopCriteria:
+    """
+    Stopping criteria for the dataset samplers.
```

Comment:
When @ramanishsingh needs to use this, or when we clean up docs, let's see if we can figure out a nice way to structure this so that we don't need to repeat / maintain multiple copies of similar docstrings